File: Src/Icm.Core.Tests/Search and Replace/PlainStringGeneratorTest.vb
Imports Icm.Text

<TestFixture(), Category("Icm")>
Public Class PlainStringGeneratorTest

    <Test()>
    Public Sub PlainStringGenerator_Test()
        Dim s As String = "HOLA"
        Dim target As New PlainStringGenerator(s)

        Dim i = 0

        For Each element In target
            Assert.That(element, [Is].EqualTo(s))
            i += 1
            If i = 20 Then Exit For
        Next
    End Sub


End Class

File: Src/Icm.Core.Tests/Search and Replace/PlainStringGeneratorTest.cs

using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using Icm.Text;

[TestFixture(), Category("Icm")]
public class PlainStringGeneratorTest
{

	[Test()]
	public void PlainStringGenerator_Test()
	{
		string s = "HOLA";
		PlainStringGenerator target = new PlainStringGenerator(s);

		dynamic i = 0;

		foreach (void element_loopVariable in target) {
			element = element_loopVariable;
			Assert.That(element, Is.EqualTo(s));
			i += 1;
			if (i == 20)
				break; // TODO: might not be correct. Was : Exit For
		}
	}


}

//=======================================================
//Service provided by Telerik (www.telerik.com)
//Conversion powered by NRefactory.
//Twitter: @telerik
//Facebook: facebook.com/telerik
//=======================================================
